source/community.md

```diff
@@ -22,6 +22,7 @@ complementary_content: |-
   by Wynn Netherland, Natalie Weizenbaum, Chris Eppstein, Brandon Mathis
 complementary_content_nav: True
 complementary_content_aria_label: 'Resources'
+no_container: false
```

Comment:
Same here -- probably not needed?

Reply:
Fixed in https://github.com/oddbird/sass-site/commit/a730f6bdbb638990191146253838e39d861ac382